lib/src/abstracts/media_picker_input.dart

```diff
@@ -3,11 +3,14 @@
 // SPDX-License-Identifier: BSD-3-Clause
 
 import 'package:flutter/material.dart';
+import 'package:flutter_media_picker/src/enums/button_type.dart';
 import 'package:flutter_media_picker/src/media_result.dart';
 
 /// Abstract class for inputs used by [MediaPicker].
```

Comment:
Please add docs describing a specific function/property above that function/property

Reply:
fixed